src/attack.rs

```diff
@@ -31,7 +31,15 @@ pub struct AttackPlugin;
 
 impl Plugin for AttackPlugin {
     fn build(&self, app: &mut App) {
-        app.add_system(player_attack.run_in_state(GameState::InGame));
+        // Can't be currently converted to a ConditionSet, since (it seems that) systems inside
+        // don't have temporal methods available (e.g. after()).
+        app.add_system(player_attack.run_in_state(GameState::InGame))
+            .add_system(player_flop.run_in_state(GameState::InGame))
+            .add_system(
+                enemy_attack
+                    .run_in_state(GameState::InGame)
+                    .after("move_to_target"),
+            );
```

Comment:
Gonna merge this because I want it in now/I am touching this code now too, but may return in future to look at this plugin/conditionset thing and see if its possible to set it up with loopless

Reply:
Sure! If/once we split in stages, the `after` relationship won't be needed anymore, since it will be implicit in the stages ordering.